guidance/library/_json.py

```diff
@@ -55,12 +57,14 @@ class Keyword(str, Enum):
 IGNORED_KEYS = {
     "$schema",
     "$id",
+    "id",
     "$comment",
     "title",
     "description",
     "default",
     "examples",
     "required",  # TODO: implement and remove from ignored list
+    "discriminator", # TODO: alternatively we could implement this in a stateful way
```

Comment:
MIght want to add a link to:
https://json-schema.org/blog/posts/validating-openapi-and-json-schema
which mentions how `discriminator` is part of the OpenAPI dialect of JSON-Schema.

Reply:
Done. Additionally added a longer comment about how disambiguating the grammar on this field could possibly improve performance or quality.

Reply:
If you have a discriminator, oneOf is equivalent to anyOf (since options are exclusive).

Reply:
Nice observation and lets us more selectively warn the user. Might have to check that the discriminator is actually unique, but otoh maybe it's just up to the user to actually give us a valid schema...